esx_service/cli/vmdkops_admin.py

```diff
@@ -304,10 +307,13 @@ def commands():
                                     'help': "Datastore which access is controlled",
                                     'required': True
                                 },
-                                '--rights': {
-                                    'help': 'Datastore access Permissions granted',
-                                    'choices': ['create', 'delete', 'mount', 'all'],
-                                    'metavar': 'create,delete,mount'
+                                '--switch-default-datastore': {
+                                    'help': "Switch default datastore for this tenant",
```

Comment:
It does not really answer the questions. Why does it need "switch" in the name ? It is confusing, inconsistent and does not really add anything. `--default-datastore` should be perfectly fine.